zerver/openapi/zulip.yaml

```diff
@@ -10630,6 +10669,201 @@ paths:
+                              Enable email notifications for new logins to account.
+                          message_content_in_email_notifications:
+                            type: boolean
+                            description: |
+                              Include the message's content in email notifications for new messages.
+                          pm_content_in_desktop_notifications:
+                            type: boolean
+                            description: |
+                              Include content of private messages in desktop notifications.
+                          wildcard_mentions_notify:
+                            type: boolean
+                            description: |
+                              Whether wildcard mentions (E.g. @**all**) should send notifications
+                              like a personal mention.
+                          desktop_icon_count_display:
+                            type: integer
+                            description: |
+                              Unread count summary (appears in desktop sidebar and browser tab)
+
+                              * 1 - All unreads
+                              * 2 - Private messages and mentions
+                              * 3 - None
+                          realm_name_in_notifications:
+                            type: boolean
+                            description: |
+                              Include organization name in subject of message notification emails.
+                          presence_enabled:
+                            type: boolean
+                            description: |
+                              Display the presence status to other users when online.
+                          enter_sends:
+                            type: boolean
+                            description: |
+                              Whether the user setting for [sending on pressing Enter](/help/enable-enter-to-send)
+                              in the compose box is enabled.
+                          enable_drafts_synchronization:
+                            type: boolean
+                            description: |
+                              A boolean parameter to control whether synchronizing drafts is enabled for
+                              the user. When synchronization is disabled, all drafts stored in the server
+                              will be automatically deleted from the server.
+
+                              This does not do anything (like sending events) to delete local copies of
+                              drafts stored in clients.
+                          email_notifications_batching_period_seconds:
+                            type: integer
+                            description: |
+                              The duration (in seconds) for which the server should wait to batch
+                              email notifications before sending them.
```

Comment:
These are duplicated as `user_settings` has all these. I cannot figure out how to deduplicate this.

Reply:
This is OK for now but clearly something we'll want to fix since it's a lot of duplicated content.  Can you open an issue for deduplicating the settings definitions in the API docs, and a second for deduplicating the `ignored_parameters_unsupported` text in the API docs (which is small but something we're planning to add to several more API endpoints in the future); you can mention @MSurfer20 on both issues since he's likely to find them fairly quick.